contracts/base/DxMath.sol

```diff
@@ -0,0 +1,102 @@
+pragma solidity ^0.4.11;
+
+contract DxMath {
```

Comment:
Hm... not 100 percent sure, but my intuition would declare it as a library.

Reply:
Yes, I agree, I also thought about that. But this would modify the contract, right? I didn't want to alter the audit.


I think this and many other things of the DutchX should be done in a more modular way so we split the code. I'll write this into the document for improvements.

We can open a branch for this kind of changes, I think is good. What do you think?